design/ceph/object/store.md

```diff
@@ -73,7 +74,124 @@ If `preservePoolsOnDelete` is set to 'true' the pools used to support the object
+
+```yaml
+  radosNamespaces:
+    metadataPoolName: rgw-meta-pool
+    dataPoolName: rgw-data-pool
+```
+
+To create the pools that will be shared by multiple object stores, create the following CephBlockPool CRs:
+
+```yaml
+apiVersion: ceph.rook.io/v1
+kind: CephBlockPool
+metadata:
+    name: rgw-meta-pool
+  spec:
+    failureDomain: host
+    replicated:
+      size: 3
+    parameters:
+      pg_num: 8
+    application: rgw
+---
+apiVersion: ceph.rook.io/v1
+kind: CephBlockPool
+ metadata:
+    name: rgw-data-pool
+  spec:
+    failureDomain: osd
+      erasureCoded:
+        dataChunks: 6
+        codingChunks: 2
+    application: rgw
+```
+
+The `.rgw.root` pool must be defined before the object store is created. This pool is used for storing the root info for all object stores.
+
+```yaml
+apiVersion: ceph.rook.io/v1
+kind: CephBlockPool
+metadata:
+    name: rgw-root
+  spec:
+    name: .rgw.root
+    failureDomain: host
+    replicated:
+      size: 3
+    parameters:
+      pg_num: 8
+    application: rgw
+```
+
+The pools for this configuration will be created as below:
+
+```bash
+# ceph osd pool ls
+.rgw.root
+rgw-meta-pool
+rgw-data-pool
+```
+
+And the pool configuration in zone is as below:
+
+```json
+# radosgw-admin zone get --rgw-zone=my-store
+{
+    "id": "2220eb5f-2751-4a51-9c7d-da4ce1b0e4e1",
+    "name": "my-store",
+    "domain_root": "rgw-meta-pool:my-store.root",
+    "control_pool": "rgw-meta-pool:my-store.control",
+    "gc_pool": "rgw-meta-pool:my-store.gc",
+    "lc_pool": "rgw-meta-pool:my-store.lc",
+    "log_pool": "rgw-meta-pool:my-store",
+    "intent_log_pool": "rgw-meta-pool:my-store.intent",
+    "usage_log_pool": "rgw-meta-pool:my-store.usage",
+    "roles_pool": "rgw-meta-pool:my-store.roles",
+    "reshard_pool": "rgw-meta-pool:my-store.reshard",
+    "user_keys_pool": "rgw-meta-pool:my-store.users.keys",
+    "user_email_pool": "rgw-meta-pool:my-store.users.email",
+    "user_swift_pool": "rgw-meta-pool:my-store.users.swift",
+    "user_uid_pool": "rgw-meta-pool:my-store.users.uid",
+    "otp_pool": "rgw-meta-pool:my-store.otp",
+    "system_key": {
+        "access_key": "",
+        "secret_key": ""
+    },
+    "placement_pools": [
+        {
+            "key": "default-placement",
+            "val": {
+                "index_pool": "rgw-data-pool:my-store.buckets.index",
+                "storage_classes": {
+                    "STANDARD": {
+                        "data_pool": "rgw-data-pool:my-store.buckets.data"
+                    }
+                },
+                "data_extra_pool": "rgw-data-pool:my-store.buckets.non-ec",
```

Comment:
Non-ec goes with data and not metadata?

Rook currently treats the index as a meta pool: https://github.com/rook/rook/blob/719a8cdb50712115eece923db3bbe724ffbe7c8d/pkg/operator/ceph/object/objectstore.go#L59-L67

Is this a bug in Rook's current implementation? Is this something that some users might want more control over? Might some users want this to be a data pool?